core/primer.md

```diff
@@ -2,6 +2,185 @@
 
 <!-- no verify-specs -->
 
+## Abstract
+
+This non-normative document provides an overview of the xRegistry specification.
+It is meant to complement the xRegistry specification to provide additional
+background and insight into the history and design decisions made during the
+development of the specification. This allows the specification itself to focus
+on the normative technical details.
+
+## Table of Contents
+
+- [History](#history)
+- [Motivation](#motivation)
+- [Design Goals](#design-goals)
+- [Representations](#representations)
+- [Possible Use Cases](#possible-use-cases)
+- [Design decisions or topics of
+  interest](#design-decisions-or-topics-of-interest)
+
+## History
+
+The CNCF Serverless Working group was originally created by the CNCF's Technical
+Oversight Committee to investigate Serverless Technology and to recommend some
+possible next steps for some CNCF related activities in this space. After
+creating the common event format [CloudEvents](https://github.com/cloudevents)
+as a foundation for an interoperable ecosystem for event-driven applications the
+next step was to create a metadata model for declaring CloudEvents, their
+payloads and associating those declarations with application endpoints. As a
+result, the xRegistry (extensible registry) specification was created.
+
+xRegistry was initially part of CloudEvents, called "CloudEvents Discovery" but
+later moved into it's own repository and CNCF project.
+
+## Motivation
+
+While CloudEvents are harmonizing different event structures across event
+providers and increase interoperability between different brokers and their
+protocols, they do not address:
+
+- **Discovery:** Where to produce/consume events? What endpoints exist?
+- **Validation:** How to validate event structures?
+- **Versioning:** How to version events – and see which versions exist?
```

Comment:
I wonder if we need to reword this slightly since we don't really "version events" as much as we "version the metadata describing the events" (e.g. their schema). It's almost the same, but I wouldn't want people to think that we're versioning the instances of events.

We should probably add something about how to figure out events are serialized - their envelope (if any and formats (xml, json & schema).

Reply:
I rephrased Versioning and added Serialization, does that match your expectations?

Reply:
yup that works - thanks